dpiprobe.go

```diff
@@ -446,7 +450,7 @@ func runTrace(
 			}
 
 			if icmpPacket != nil {
-				fmt.Printf("%s %s\n", ipPacket.SrcIP, ipSourceDnsNameFragment)
+				fmt.Printf("%s %s %s \n", ipPacket.SrcIP, ipSourceDnsNameFragment, elapsedTime)
```

Comment:
The format parameter of ipSourceDnsNameFragment (the second %s) is replaced by the DNS name plus a space, so don't leave a space after it: %s%s.